openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -1169,3 +1171,53 @@ def _replace_username_for_all_models(self, current_username, new_username, repla
                 new_username,
             )
         return True
+
+
+class ThirdPartyAuthStatusView(APIView):
```

Comment:
Can we move this view and API to the `third_party_auth` django app?   There is already [an endpoint there](https://github.com/edx/edx-platform/blob/e712be7892e10263d1dfdf4b4fd4c867d0f17fa4/common/djangoapps/third_party_auth/api/views.py#L200) that lists the 3PA backends linked to a user account.  Also, it allows us to keep the domain logic of 3PA (in terms of manipulating the pipeline methods) to be within that app.